src/index.js

```diff
@@ -60,9 +61,24 @@ io.on('connection', function(socket) {
 
     if (hash === doc.hash) {
       user = doc;
-      return done(success({name: user.name}));
+      return done(success({name: user.name, bio: user.bio}));
     }
 
     return done(error('wrong username/password'));
   });
+
+  socket.on('cl_update', async ({name, newName, newBio}, done) => {
+    debug('cl_update', name, newName, newBio);
+
+    const doc = await User.findOne({name});
```

Comment:
Do not use `User.findOne()` since the `user` is already in the scope.
Also, need to check for login.